_component_design/popovers.md

```diff
@@ -14,81 +14,171 @@ last-modified: 2017-08-17
-come in a the following types Hover Popover, Triggered Popover, and Closeable
-Popover
+## Introduction
+{: .hxSectionTitle}
+<div class="hxRow">
 
-# Hover Popover
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to Use
+{: .hxSubSectionTitle}
+Popovers are triggered as a result of a user action and provide a user with additional context through language, actions through buttons, or both. 
+
+Popovers may also be used for short create/edit workflows, and are particularly useful if context from the page is needed while making the change. If a large workflow is required, consider creating a new page.
 
-<div class="hxRow">
-{% column left:"hxCol-4" %}
-Hover popovers are triggered as a result of a user hovering over linked area in
-the UI.
 
-- font-family: Roboto
-- font-weight: 400
-- font-size: 16px / 1em
-- color: Grey800 (#616161)
+### Best Practice
+{: .hxSubSectionTitle}
+
+- Avoid opening a popover from within another popover
+- Open popovers via click, and never open them via hover.
+- Avoid putting important or required information in a popover because the content inside of a popover isn't always visible.
+- Use popovers to make small changes, with a small number of inputs.
+
+### Problems Being Solved
+{: .hxSubSectionTitle}
+- For users needing to make changes or take actions while remaining in their original context (page) a popover allows for small changes or actions to take place without leaving the page.
+- For user needing to be able to view additional details about an item in a workflow, a popover allows easy access to this complex or interactive information without cluttering the page.
 
 {% endcolumn %}
-{% column right:"hxCol-8" %}
-{% figure [caption:"Hover popover"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-hover-specs.svg){:width="100%"}
-{% endfigure %}
-{% figure [caption:"Hover popover specs"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-hover-specs2.svg){:width="100%"}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Meet the popover pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-hero-image.svg){:width="100%"}
 {% endfigure %}
 {% endcolumn %}
-</div>
 
-## Triggered Popover
+</div>
 
+## Default Layout 
+{: .hxSectionTitle}
 <div class="hxRow">
-{% column left:"hxCol-4" %}
-Overpopovers are triggered as a result of a user hovering over linked area in
-the UI.
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-- font-family: Roboto
-- font-weight: 400
-- font-size: 16px / 1em
-- color: Grey800 (#616161)
+The most defining aspects of a popover are the container and the pointer. The container  appears on top of the canvas. The pointer points to the popover trigger. Pointers may be used on either side of the four corners.
```

Comment:
Editorial: Remove space between `container` and `appears`

Reply:
removed.